elements/pl-file-upload/pl-file-upload.py

```diff
@@ -2,42 +2,40 @@
 import hashlib
 import json
 from io import StringIO
+from typing import List
 
 import chevron
 import lxml.html
 import prairielearn as pl
 
 
-def get_file_names_as_array(raw_file_names):
-    raw_file_names = StringIO(raw_file_names)
+def get_file_names_as_array(raw_file_names: str) -> List[str]:
     reader = csv.reader(
-        raw_file_names,
+        StringIO(raw_file_names),
         delimiter=",",
         escapechar="\\",
         quoting=csv.QUOTE_NONE,
         skipinitialspace=True,
         strict=True,
     )
-    for row in reader:
-        # Assume only one row
-        return row
+    return next(reader)
 
 
 # Each pl-file-upload element is uniquely identified by the SHA1 hash of its
 # file_names attribute
-def get_answer_name(file_names):
+def get_answer_name(file_names: str) -> str:
     return "_file_upload_{0}".format(
         hashlib.sha1(file_names.encode("utf-8")).hexdigest()
     )
 
 
-def add_format_error(data, error_string):
+def add_format_error(data: pl.QuestionData, error_string: str):
```

Comment:
Functions that don't return anything need a `None` return annotation. Otherwise, the typechecker assumes the return type to be `Any` (in other words, doesn't enforce any behavior).
```suggestion
def add_format_error(data: pl.QuestionData, error_string: str) -> None:
```